launcher/src/test/java/org/wildfly/core/launcher/CommandBuilderTest.java

```diff
@@ -273,9 +273,11 @@ private void testJPMSArguments(final Collection<String> command, final int expec
         assertArgumentExists(command, "--add-exports=java.naming/com.sun.jndi.url.ldap=ALL-UNNAMED", expectedCount);
         assertArgumentExists(command, "--add-exports=java.naming/com.sun.jndi.url.ldaps=ALL-UNNAMED", expectedCount);
         assertArgumentExists(command, "--add-exports=jdk.naming.dns/com.sun.jndi.dns=ALL-UNNAMED", expectedCount);
+        assertArgumentExists(command, "--add-opens=java.base/java.lang=ALL-UNNAMED", expectedCount);
```

Comment:
Seems as out of scope of this change.

Double checking if this is intentional.

Reply:
I think this is fine; this test is testing that the expected JVM modular options are also used when launching jar files or the server for the tests

Reply:
Yeah, this was just a missing assert that Rio took the opportunity to correct.